.github/workflows/cypress-tests.yaml

```diff
@@ -0,0 +1,51 @@
+name: Cypress tests
+on:
+  pull_request:
+    paths:
+      - 'packages/ui/**/*'
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    container: cypress/browsers:node-20.5.0-chrome-114.0.5735.133-1-ff-114.0.2-edge-114.0.1823.51-1
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
there are already 3 version for quite a while, so I would suggest to update it

Reply:
sure, let's try.

Reply:
done